Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/runtime/helper.go

```diff
@@ -61,11 +65,16 @@ func ExtractList(obj Object) ([]Object, error) {
 	list := make([]Object, items.Len())
 	for i := range list {
 		raw := items.Index(i)
-		item, ok := raw.Addr().Interface().(Object)
-		if !ok {
+		var found bool
+		switch raw.Kind() {
+		case reflect.Interface:
```

Comment:
@smarterclayton so this works, I did the same to GetItemsPtr just to make sure it works. I also updated the test to exercise 'generic' list as well. Going to submit upstream PR and will update this commit message to include upstream commit ID.